todo

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+source /todo/.todorc
```

Comment:
`.todorc` file should be in the home directory of the user that's running the command